io.md

```diff
@@ -89,17 +89,15 @@ So far, when we have been writing and using strings, or reading and writing to a
 <class 'str'>
 ```
 
-When we read or write to a file or when we talk to other computers on the Internet, we need to convert our unicode strings into a format that can be sent and received, and that format is called "UTF-8". We can read and write in that format, using a simple keyword argument to our standard `open` function:
+The rules for translating Unicode into bytes (which is what computers use to process info) is called encoding.  Encoding is important when you want to send and receive info from other computers, and a popular encoding to use is UTF-8.  We can read and write in UTF-8 by using a simple keyword argument in our `open function`.
 
 <pre><code class="lang-python">{% include "./programs/io_unicode.py" %}</code></pre>
 
 **How It Works**
 
-You can ignore the `import` statement for now, we'll explore that in detail in the [modules chapter](./modules.md#modules).
+We use  io.open  and then use the `encoding` argument in the first open statement to encode the message, and then again in the second open statement when decoding the message.  Note that we should only use encoding in the open statement when in text mode.
```

Comment:
Why remove the sentence about `import`?

Reply:
Because you covered it much earlier in your book in the Modules section and used it multiple times.  I figured the reader would be familiar with it, but totally up to you.

Reply:
Aah got it.